server/src/main/java/io/spine/server/model/MethodSignature.java

```diff
@@ -202,7 +202,7 @@ public final Class<? extends Annotation> annotation() {
     }
 
     /**
-     * Creates a {@linkplain HandlerMethod handler method} from a raw method, if the passed
+     * Creates a {@linkplain Receptor handler method} from a raw method, if the passed
```

Comment:
Not sure whether we want to double it as "handler method", and not a "receptor". At least, you had this change in a file below, but not here.